### PR TITLE
Consolidated sections

### PR DESCRIPTION
**Pull request by jerae-duffin**
Consolidated Pro and Team section.

### CODE REVIEW

File: subscription/upgrade.md [hunk 1]
```diff
@@ -11,19 +11,19 @@ This page contains information on how to purchase a Docker paid subscription. It
...
 
-2. Click **Upgrade** and select your Pro, Team, or Business subscription.
+2. To upgrade your Personal subscription, click **Upgrade**. To upgrade your Pro or Team subscription, open the drop-down menu next to your username in the top-right corner and select **Billing**. On the **Billing Details** page, click **Change plan** to navigate to the pricing page.
 
    > **Note**
    >
-   > If you have not created an organization for a Team or Business subscription, select **Create Organization** from the drop-down menu and select your plan. Click **Buy Now** and continue with the on-screen instructions.
+   > If you have not created an organization for a Team or Business subscription, select **Create Organization** from the drop-down menu and select your plan. If you already have a subscription, you can also add another organization. Click **Buy Now** and continue with the on-screen instructions.
 
 3. On the **Organization Size** page, select your annual or monthly subscription and update the number of users for your organization. For a Business subscription, the only option available is annually.
```
Comment (usha-mandya) [inline subscription/upgrade.md:28]: Suggest reword:



>For a Business subscription, the only option available is annually.

Here's another suggestion. Let me know what you think 
 
"If you are upgrading to a Business subscription, you must choose an annual subscription."

File: subscription/upgrade.md [hunk 2]
```diff
@@ -32,41 +32,7 @@ To purchase a Pro, Team, or Business subscription:
     >
     > To upgrade to a Business subscription, you must have at least 5 users and at least 40 users through purchase orders. To create a purchase order,  specify the number of users and then click the **Contact Sales form**. Then, continue the on-screen instructions for the billing details and click **Purchase**.
```
Comment (usha-mandya) [inline subscription/upgrade.md:33]:
```suggestion
    > To upgrade to a Business subscription, you must have at least 5 users and at least 40 users for purchase orders. To create a purchase order, specify the number of users and then click the **Contact Sales form**. Then, continue the on-screen instructions for the billing details and click **Purchase**.
```

File: subscription/upgrade.md [hunk 3]
```diff
@@ -11,19 +11,19 @@ This page contains information on how to purchase a Docker paid subscription. It
 
 You can upgrade from Docker Personal to a Pro, Team, or a Business subscription from a Personal subscription. When you upgrade to a paid subscription, you can immediately use all the features and entitlements offered in your new subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
 
-## Upgrade from a Personal to a Pro, Team, or Business subscription
+## Upgrade to a Pro, Team, or Business subscription
 
 The following section contains information on how to purchase a Docker  subscription for new customers. To upgrade from a Personal subscription to a Team or Business subscription, you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply subscription/upgrade.md:16]: @usha-mandya Only Personal, because Pro may have an organization already associated with it.

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**